lib/manifest.go

```diff
@@ -76,6 +79,9 @@ func ManifestsFromFiles(files []string) ([]Manifest, error) {
 			errs = append(errs, errors.Wrapf(err, "error parsing %s", file.Name()))
 			continue
 		}
+		for _, m := range ms {
```

Comment:
nit: so a bunch of manifests will have the same name... can we add that as comment to `Name` field. please do not use it to dedup or as a map key.